gui/src/main/java/de/azubi/spiele/gui/fx/spielbrett/GUISpielbrett.java

```diff
@@ -46,45 +50,46 @@
     public Spieler spielerDran;
 
     public void wuerfeln(ActionEvent actionEvent) {
-        System.out.println(spielerDran.isDarfDreimalWuerfeln());
-        if (spielerDran.isDarfNochWuerfeln()) {
-            int gewuerfelt = (int) (Math.random() * 6) + 1;
-            lblWuerfeln.setText("Du hast eine " + gewuerfelt + " gewürfelt!");
-            lblWuerfeln.setVisible(true);
-            spielManager.setZahlGewuerfelt(gewuerfelt);
-            zahlGewuerfelt = gewuerfelt;
-            anzahlWuerfe++;
-
-            System.out.println(anzahlWuerfe);
-
-            if (gewuerfelt == 6 && !spielManager.eigeneFigurBereitsAufFeld(spielerDran)) {
-                if (spielerDran.getHaus().getEnthalteneFiguren().size() > 0) {
-                    spielManager.figurZiehen(spielerDran, gewuerfelt);
-                    setIcon(removeFigureFromHouse(), "blank");
-                    setIcon(getSpielerStart(), spielerDran.getFarbe());
-                    ausHaus = true;
-                    spielerDran.setDarfDreimalWuerfeln(false);
-                    lastNumber = 6;
+        if (!spielManager.isBeendet()) {
```

Comment:
Das ist recht viel, was da in der if-Anweisung steht.
Hier wäre es schöner/besser Lesbar/sicherer das in ein, zwei Methoden auszulagern.